lab-davide/test/test.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+const main = require(__dirname + '/index');
```

Comment:
This require statement is broken, __dirname points at the test folder, not the root folder of the directory that index is in.